.idea/copyright/profiles_settings.xml

```diff
@@ -0,0 +1,3 @@
+<component name="CopyrightManager">
```

Comment:
IMO this doesn't need to be in source control. A shared gradle project should have no IDE specific preferences stored on VCS unless its a must have setting to build the project.